### ISSUE TITLE
bug(snackbar): JAWS does not voice Snackbar content after modal close

### ISSUE THREAD

**Issue by dwaynelavon** [angular/components#21707]
#### Reproduction

Stackblitz: https://stackblitz.com/edit/angular-bat55z?file=src/app/dialog-content-example.ts

Steps to reproduce:
1. Activate JAWS
2. Visit Stackblitz link
3. Click "Open dialog" button to open the dialog.
4. Close the dialog
5. Observe JAWS voicing. 
 
 
#### Expected Behavior

What behavior were you expecting to see?
JAWS does not voice Snackbar content

#### Actual Behavior

What behavior did you actually see?
JAWS should voice Snackbar content as it appears on the screen. Seems to only happen with JAWS in this scenario and works fine if dialogs are not involved.

#### Environment

- Angular: 11.0.3
- CDK/Material: 11.0.3
- Browser(s): Firefox
- Operating System (e.g. Windows, macOS, Ubuntu): Windows with JAWS

**Comment by crisbeto** [comment]
It might take a while for us to get back to this, because we don't have access to JAWS while everybody is WFH. See https://github.com/angular/components/issues/19541#issuecomment-639671264.

**Comment by zarend** [comment]
Hello @dwaynelavon ,

Thank you for reporting. I believe we may have fixed this issue in v15. I was wondering if we could check with you if this issue requires further action. Does upgrading to v15 resolve the issue for you?

Thanks,
Zach

**Comment by zarend** [comment]
Hello @dwaynelavon ,

Thank you for reporting this issue. Unfortunately we are not able to take action if we cannot reproduce it. This might have already been fixed. If the issue persists, please provide reproduction steps or file a new issue report.

- Zach
